mix.exs

```diff
@@ -62,9 +62,7 @@ defmodule Screenplay.MixProject do
       {:stream_data, "~> 1.1", only: :test},
       {:ecto_sql, "~> 3.0"},
       {:postgrex, ">= 0.0.0"},
-      {:screens_config,
-       git: "https://github.com/mbta/screens-config-lib.git",
-       ref: "c0db78ecd43c633a55a1f24b268ae7b8a5214748"},
+      {:screens_config, github: "mbta/screens-config-lib", branch: "cfg-un-v2"},
```

Comment:
Is the general flow of changes in the `screens-config` library to merge something in there and then put up PRs for deps/lock file changes in the respective repositories?

Reply:
Usually I put up PRs for config-lib/Screens/Screenplay all concurrently since it allows running CI for Screens/Screenplay on the config-lib changes and confirming that won't cause any problems. I could have also kept the Screens/Screenplay PRs in draft until the config-lib changes were approved and merged, but in this case it seemed more efficient to do it this way from a reviewing perspective. Now that those changes _are_ merged, I can update these two PRs and remove the branch specifier before merging them.